pipelines/epinow2/generate_config.py

```diff
@@ -1 +1,44 @@
-pass
+import os
+from datetime import date, datetime
+
+from utils.epinow2.functions import (
+    generate_job_id,
+    generate_task_configs,
+    validate_args,
+)
+
+if __name__ == "__main__":
+    """
+    A script to generate `epinow2` configuration objects. This
+    will be invoked either by 1) a scheduled GH action using
+    default values, or 2) a user-triggered workflow with supplied
+    state, pathogen, report_date, and reference_date parameters.
+    This script is the entrypoint to the workflow that generates
+    configuration objects, validates them against a schema, and
+    writes them to Blob Storage.
+    """
+
+    # Pull run parameters from environment
+    state = os.environ.get("state", "all")
+    pathogen = os.environ.get("pathogen", "all")
+    report_date = os.environ.get("report_date", date.today())
+    reference_dates = os.environ.get("reference_date", [report_date])
```

Comment:
Do we want to pass a list of reference dates or just a min and a max?

Reply:
If we do min/max would we need to generate a list of intermediate days or does the model handle this? (eg if we're passed [2024-01-01, 2024-01-03], should we parse this to --> [2024-01-01, 2024-01-02, 2024-01-03]?)

Reply:
I ended up setting the data reader to take a [min/max](https://github.com/CDCgov/cfa-epinow2-pipeline/blob/b859765825e0eb16c4894943dc9a0c92fd763839/R/read_data.R#L29) and throw a warning if any reference dates in that period are missing.

Reply:
Warning is [here](https://github.com/CDCgov/cfa-epinow2-pipeline/blob/b859765825e0eb16c4894943dc9a0c92fd763839/R/read_data.R#L153)